repository/git.go

```diff
@@ -809,179 +1056,133 @@ func (repo *GitRepo) PullNotes(remote, notesRefPattern string) error {
 	return repo.MergeNotes(remote, notesRefPattern)
 }
 
-func getRemoteArchiveRef(remote, archiveRefPattern string) string {
-	relativeArchiveRef := strings.TrimPrefix(archiveRefPattern, "refs/devtools/archives/")
-	return "refs/devtools/remoteArchives/" + remote + "/" + relativeArchiveRef
+func getRemoteDevtoolsRef(remote, devtoolsRefPattern string) string {
+	relativeRef := strings.TrimPrefix(devtoolsRefPattern, "refs/devtools/")
```

Comment:
can you reference the var `devtoolsRefPrefix` instead of inlining the string here?

Reply:
Done